lib/network.dart

```
@@ -388,14 +392,14 @@ class FetchStrategyBuilder {
 
   /// Uses [defaultTransientHttpStatusCodes] to determine if the [statusCode] is
   /// transient.
-  static bool defaultTransientHttpStatusCodePredicate(int statusCode) {
+  static bool defaultTransientHttpStatusCodePredicate(int? statusCode) {
```

Making this nullable will cause a null value to silently return false, which is almost certainly not going to be what the calling code intended. This should be non-nullable, and the caller should be handling null.

---

Sure.